packages/grafana-data/src/types/panel.ts

```diff
@@ -26,6 +27,8 @@ export interface PanelPluginMeta extends PluginMeta {
   hideFromList?: boolean;
   /** Sort order */
   sort: number;
+
+  visualizationType?: string;
```

Comment:
@academo You mentioned you would like this to be prefixed by explore. I argumented that this may be used in other context and should not be named based on current usage as that may change and would no longer reflect what it is. Let me know if you still think this needs a prefix (same naming will be in the plugins.json).